README.md

```diff
@@ -1,62 +1,109 @@
-Photoshop.d.ts
-==============
+# Photoshop.d.ts
 
 [![npm version](https://badge.fury.io/js/photoshop.d.ts.svg)](http://badge.fury.io/js/photoshop.d.ts)
 [![Build Status](https://travis-ci.org/felixSchl/photoshop.d.ts.svg)](https://travis-ci.org/felixSchl/photoshop.d.ts)
 ![dependencies status](https://david-dm.org/felixschl/photoshop.d.ts.svg)
 
-> Type-safe Photoshop scripting
+**Type-safe Photoshop scripting**
+
+This project aims to create and maintain TypeScript declaration files for Adobe ExtendScript, focussed on Photoshop. It enables you writing Photoshop scripts with the power and safety of TypeScript.
```

Comment:
A nit pick, but let's keep these lines under 80 characters so it looks good on the terminal when viewed via `less` and for those of us using command line editors... Also: "It enables you writing Photoshop scripts" should probably read "It enables you to write Photoshop scripts"